src/seth/libexec/seth/seth

```diff
@@ -145,7 +150,9 @@ while [[ $1 ]]; do
        --state)      shift; export HEVM_STATE=$1;;
        --timestamp)  shift; export HEVM_TIMESTAMP=$1;;
        --debug)             export HEVM_DEBUG=yes;;
+       --hevm)              export HEVM_EXEC=yes;;
        --source)     shift; export DAPP_JSON=$1;;
+       --code)       shift; export HEVM_CODE=$1;;
```

Comment:
this is also a flag to `seth call`? Will you add it to the options above as well? This would load the $TO contract and then overwrite its code, iiuc. This is pretty esoteric, but maybe cool to allow

Reply:
Yes, will add docs.

Reply:
One use would be for becoming the root user of some system, say you overwrote the governance contract. Another would be testing out some different mechanism somewhere that is heavily integrated with internal/external contracts that you don't want to replicate.